### PR TITLE
rearrange stuff to match MinigameCoreAPI setup

### PR DESCRIPTION
**Pull request by kinggoesgaming**
gradle and other stufff

### CODE REVIEW

**Comment by kinggoesgaming** [comment]
i think i derped a bit fixing the issues

**Comment by kinggoesgaming** [comment]
retargeting bleeding

**Comment by Flibio** [comment]
Can you please squash into one commit entitled `Gradle Improvements`?

**Comment by kinggoesgaming** [comment]
I am away from computer atm will when I do get the time

**Comment by kinggoesgaming** [comment]
I decided to name the squashed commit `backend changes` instead cause it has many non gradle changes aswell

**Comment by kinggoesgaming** [comment]
good to go

Just make travis required for master

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,8 @@
 name=MinigameCore
-owner=Flibio
-inceptionYear=2015
+organization=MinigameCore
 currentYear=2016
+description=The Minigame Core framework.
```
Comment (Flibio) [inline gradle.properties]: No space in MinigameCore here.

File: scripts/pre-commit [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+# -u = --no-search-upwards
+./gradlew --daemon -u licenseFormat spotlessApply
```
Comment (Flibio) [inline scripts/pre-commit]: Newline.

**Comment by Flibio** [comment]
Approved.

**Pull request merged**